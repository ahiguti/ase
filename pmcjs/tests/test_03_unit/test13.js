
Perl.Use("TieTest");
var fs = Perl.Functions["TieTest"];
var x = fs.tietest(1);
x['a'] = 5;
x['b'] = 6;
x['c'] = 7;
print("xb=" + x['b']);
delete x['c'];
for (var i in x) {
  print("k=" + i + " v=" + x[i]);
}
