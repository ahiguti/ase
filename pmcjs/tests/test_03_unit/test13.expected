tiehash store [a] [5]
tiehash fetch [a]
tiehash store [b] [6]
tiehash fetch [b]
tiehash store [c] [7]
tiehash fetch [c]
tiehash fetch [b]
tiehash delete [c]
tiehash firstkey = a
tiehash nextkey = b
tiehash nextkey = 
tiehash fetch [a]
tiehash fetch [b]
xb=6
k=a v=5
k=b v=6
